REVERT this upstream patch which has broken builds on older systems
and also on current systems that include this header
kencu@macports.org

https://git.xiph.org/?p=ogg.git;a=commitdiff;h=6449883ccacfee276ed9d99fa047342cdc51ab88

From: Brion Vibber <brion@pobox.com>
Date: Mon, 29 Jan 2018 21:52:20 +0000 (-0800)
Subject: Fix iOS/macOS build with current SDKs.
X-Git-Tag: v1.3.4~21
X-Git-Url: https://git.xiph.org/?p=ogg.git;a=commitdiff_plain;h=6449883ccacfee276ed9d99fa047342cdc51ab88

Fix iOS/macOS build with current SDKs.

Use <sys/types.h> not <inttypes.h> when building for Apple platforms,
as it works with modular dynamic-framework builds.

Signed-off-by: Ralph Giles <giles@thaumas.net>
---

diff --git include/ogg/os_types.h include/ogg/os_types.h
index 4165bce..eb8a322 100644
--- include/ogg/os_types.h
+++ include/ogg/os_types.h
@@ -70,7 +70,7 @@
 
 #elif (defined(__APPLE__) && defined(__MACH__)) /* MacOS X Framework build */
 
-#  include <sys/types.h>
+#  include <inttypes.h>
    typedef int16_t ogg_int16_t;
    typedef uint16_t ogg_uint16_t;
    typedef int32_t ogg_int32_t;
